cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -508,7 +512,74 @@ In order to configure CDAP Master for Kerberos authentication:
 
 - When CDAP Master is started, it will login using the configured keytab file and principal.
 
-**Note:** CDAP support for secure Hadoop clusters is limited to CDH 5.0 through CDH 5.3.x, and HDP 2.0 or higher.
+
+In order to configure **CDAP Explore Service for secure Hadoop:**
+
+.. highlight:: xml
+
+- To the configuration file ``core-site.xml``, set these properties::
+
+    <property>
+      <name>hadoop.proxyuser.hive.groups</name>
+      <value>cdap,hadoop,hive</value>
+    </property>
+    <property>
+      <name>hadoop.proxyuser.hive.hosts</name>
+      <value>*</value>
+    </property>
+
+- To the configuration file ``mapred-site.xml``, set these properties, substituting your
+  domain for ``EXAMPLE.COM``, and the hostname of the node running the JHS
+  (JobHistoryServer) for ``HOSTNAME_OF_JHS``::
+
+    <property>
+      <name>mapreduce.jobhistory.keytab</name>
+      <value>/etc/security/keytabs/jhs.service.keytab</value>
+    </property>
+    <property>
+      <name>mapreduce.jobhistory.principal</name>
+        <value>jhs/_HOST@EXAMPLE>.COM</value>
```

Comment:
Actually maybe not, since we have `hive/_HOST@EXAMPLE.com`. Still, there's an extra `>` in there which should be removed.